defaults/initrd.scripts

```diff
@@ -1553,6 +1571,71 @@ splash() {
 	return 0
 }
 
+splashCmd() {
```

Comment:
Why not just implement an existing empty `splash()`?

Reply:
The splash vs splashCmd stuff was mostly for trying to accommodate the deprecated fbsplash.
Rewritten in next commit to use existing splash() function, which also cleans up the changes a bit.